### PR TITLE
Fix/layout fixes

### PR DESCRIPTION
**Pull request by Yoinezero**
fix some problems with ui

### CODE REVIEW

**Comment by Yoinezero** [comment]
image
image
image
image

**Comment by ngame2011** [comment]
@IAlexHilimovich @Chichmarin17 review please

File: packages/frontapp/src/app/components/Sidebar/index.tsx [hunk 1]
```diff
@@ -49,11 +49,11 @@ const StyledSidebar = styled.aside<IStyledSidebarProps>`
   ::-webkit-scrollbar {
     width: 0;
   }
-  @media (max-width: ${dimensions.laptop_width}) {
-    max-width: 18rem;
-  }
+  // @media (max-width: ${dimensions.laptop_width}) {
+  //   max-width: 18rem;
+  // }
```
Comment (IAlexHilimovich) [inline packages/frontapp/src/app/components/Sidebar/index.tsx:54]: No need to keep commented code

File: packages/frontapp/src/app/components/UserList/SingleUser.tsx [hunk 2]
```diff
@@ -61,6 +73,8 @@ export interface ISingleUser {
   name: string;
   statuses: IClaimHistory[];
   className?: string;
+  underlined?: boolean;
+  hoverable?: boolean;
```
Comment (IAlexHilimovich) [inline packages/frontapp/src/app/components/UserList/SingleUser.tsx:77]: `hoverable` never used?

**Comment by Yoinezero** [comment]
Library page with different resolutions
image
image
image
image
image
image
image

File: packages/frontapp/src/app/components/UserList/SingleUser.tsx [hunk 3]
```diff
@@ -84,8 +99,8 @@ const SingleUser: FC<ISingleUser> = ({
       <AvatarWrapper>
         <Avatar src={avatar || mockData.avatar} />
       </AvatarWrapper>
-      <InfoWrapper>
-        <Description>{name}</Description>
+      <InfoWrapper underlined={underlined}>
+        <Description style={{ marginBottom: '0.5rem' }}>{name}</Description>
```
Comment (IAlexHilimovich) [inline packages/frontapp/src/app/components/UserList/SingleUser.tsx:102]: can move marginBottom to Description component definition

File: packages/frontapp/src/app/globalUI/Table/index.tsx [hunk 4]
```diff
@@ -20,15 +20,15 @@ const TableContainer = styled.div`
 
 const List = styled.div`
   box-sizing: border-box;
-  padding: 0 ${dimensions.xl} 0 0;
+  //padding: 0 ${dimensions.xl} 0 0;
```
Comment (IAlexHilimovich) [inline packages/frontapp/src/app/globalUI/Table/index.tsx:23]: Delete comment

**Review by IAlexHilimovich — APPROVED** [review]

**Review by IAlexHilimovich — APPROVED** [review]

File: packages/frontapp/src/app/components/BookInfo/index.tsx [hunk 5]
```diff
@@ -472,6 +465,81 @@ const BookInfo: FC<IBookInfoProps> = ({
...
+          value="Delete item"
+          transparent
+          secondary
+          svgComponent={<Remove />}
+          onClick={handleDeleteBtn}
+        />
+      </WrapperButtons>
+    );
+  };
+
```
Comment (Chichmarin17) [inline packages/frontapp/src/app/components/BookInfo/index.tsx:542]: It's a bad practice to define component inside another component, react will re-create the component each time when `BookInfo` was changed, so the `buttons` component will be unmounted, removed from the DOM and then mounted again. Better to move it out either to another function inside `index.tsx` or to a different file/module

File: packages/frontapp/src/app/components/UserList/SingleUser.tsx [hunk 6]
```diff
@@ -8,10 +8,18 @@ import { IClaimHistory } from '../../models/helperFunctions/claimHistory';
 import ClaimHistory from '../ClaimHistory';
 import { RoutesTypes } from '../../../utils/routes';
 
-const InfoWrapper = styled.div`
+interface IInfoWrapper {
+  underlined?: boolean;
```
Comment (Chichmarin17) [inline packages/frontapp/src/app/components/UserList/SingleUser.tsx:12]: isUnderlined could be better, but minor

File: packages/frontapp/src/app/components/UserList/SingleUser.tsx [hunk 7]
```diff
@@ -69,6 +83,7 @@ const SingleUser: FC<ISingleUser> = ({
   name,
   avatar,
   className,
+  underlined = false,
```
Comment (Chichmarin17) [inline packages/frontapp/src/app/components/UserList/SingleUser.tsx:86]: see above

File: packages/ui-kit/src/lib/dimensions/index.ts [hunk 8]
```diff
@@ -13,6 +13,7 @@ export const dimensions = {
   xl_6: '2.5rem', //40px
   base_3: '3rem', //48px
   xl_10: '3.125rem', //50px
+  wide_laptop_width: '85rem', //1360px
```
Comment (Chichmarin17) [inline packages/ui-kit/src/lib/dimensions/index.ts:16]: Usually, the next grade is `1440px`, but if that could break the layour we can keep it as it is

**Review by Chichmarin17 — CHANGES_REQUESTED** [review]

File: packages/frontapp/src/app/components/BookInfo/index.tsx (hunk 5 above)
Comment (Yoinezero) [reply packages/frontapp/src/app/components/BookInfo/index.tsx:542]: We use these buttons only in this component, but we have to move them from top to bottom in laptop mode.
And i don't use this function as a component, but like this `{isLaptop ? buttons() : null}`.
Which is the best way to refactor it?

Comment (Yoinezero) [reply packages/frontapp/src/app/components/BookInfo/index.tsx:542]: > It's a bad practice to define component inside another component, react will re-create the component each time when `BookInfo` was changed, so the `buttons` component will be unmounted, removed from the DOM and then mounted again. Better to move it out either to another function inside `index.tsx` or to a different file/module

We use these buttons only in this component, but we have to move them from top to bottom in laptop mode.
And i don't use this function as a component, but like this `{isLaptop ? buttons() : null}`.
Which is the best way to refactor it?

Comment (Chichmarin17) [reply packages/frontapp/src/app/components/BookInfo/index.tsx:542]: You can move it to a separate function within the file if it is the only place where you use it. Also, use functional component convention naming (`Buttons`)

File: packages/frontapp/src/app/components/BookInfo/index.tsx [hunk 9]
```diff
@@ -472,6 +464,15 @@ const BookInfo: FC<IBookInfoProps> = ({
...
 
+  console.log('Status: ', statusInfo?.status !== 'Free');
+  console.log('Id: ', statusInfo?.person_id === id);
+  console.log('Id: ', id);
+  console.log('Id: ', statusInfo?.person_id);
+  console.log(
+    'Expr: ',
+    statusInfo?.status !== 'Free' && statusInfo?.person_id === id
+  );
+
```
Comment (Chichmarin17) [inline packages/frontapp/src/app/components/BookInfo/index.tsx:475]: Let's remove `console.log`

**Review by Chichmarin17 — APPROVED** [review]

**Pull request merged**